INSTALLING.md

```diff
@@ -70,19 +70,31 @@ docker-compose up -d
 
 PostgreSQL and Redis will now set itself up in the background, and will be available in
 a minute or so.
-You'll need to run the `docker-compose up` command again if you
-reboot your computer to start the database again.
-
+You'll need to run the `docker-compose up` command again (in a Nix-shell) if you
+reboot your computer to start the database again. If you like, you can run the
+container in the background by adding `-d` to the mentioned command.
 If you're already running a copy of PostgreSQL, you can use this too.
```

Comment:
This sentence belongs together with these lines:
> You'll need to create a user with all privileges for the
databases `koala-development` and `koala-test`, and set this in the `.env` file.

which you moved down a little bit.
I guess we can remove the lines, as most people that set up their own database will know how to set up things anyhow.

Reply:
I will remove them as I, someone fairly new to all this, didn't understand it, so taking that in account too I guess that most people don't have a use for it.